planner/core/plan.go

```diff
@@ -331,6 +331,9 @@ type PhysicalPlan interface {
 	// GetPlanCost calculates the cost of the plan if it has not been calculated yet and returns the cost.
 	GetPlanCost(taskType property.TaskType, costFlag uint64) (float64, error)
 
+	// GetNetworkCost calculates the cost of the plan in network data transfer.
+	GetNetworkCost() float64
```

Comment:
This function seems not general, suggest removing it from this interface and implementing it for `IndexReader`, `TableReader`, `IndexMerge`, and `LookUp` dedicatedly.